src/Symfony/Component/Form/FormEvents.php

```diff
@@ -29,26 +29,45 @@ final class FormEvents
      *  - Change data from the request, before submitting the data to the form.
      *  - Add or remove form fields, before submitting the data to the form.
      *
+     * Any parent is being submitted during this event, it would have fired its PRE_SUBMIT
```

Comment:
Any parent **that** is being submitted during this event, **~it~** would have fired...?

Reply:
I've removed that sentence x)